### DISCUSSION TITLE
Switch to indicate alarm triggered before siren

### DISCUSSION THREAD

**Discussion by TristanBrotherton** [MiguelRipoll23/homebridge-securitysystem#295] [Ideas]
Hi there, this is a wonderful plug-in. I have setup a 30 second delay for the alarm to give me time to disarm it when entering the house. Id like to have a switch that indicates the alarm is triggered during that 30 seconds that I could use to indicate the alarm will go off if it's not disarmed. If this is possible another way Id love to know!

**Comment by TristanBrotherton** [comment]
@MiguelRipoll23 - Are you aware of any work-arounds I could make to achieve this? Essentially I want to be able to make an automation that would give a visual prompt the alarm is triggered and will go off if it is not disarmed. Something common in traditional alarms.

**Reply by MiguelRipoll23** [reply]
Hello, sorry rough week at work. Correct me if I’m wrong but you should be able to use the Siren switch for this. When the Siren switch turns on, if a trigger time is provided it will remain on until the security system is either disarmed or triggered.

**Reply by MiguelRipoll23** [reply]
If you are looking for some kind of warning before the very last seconds, maybe the "Show Siren Tripped Sensor" can help you out.

**Reply by TristanBrotherton** [reply]
Thats extremely helpful, sorry to bother you! Hope this week is not like last week.

**Reply by MiguelRipoll23** [reply]
No problem, be free to reach out if you still need help.

**Reply by TristanBrotherton** [reply]
Aha - i see the issue, I was using the siren switch to actually turn on the physical siren. Maybe that was not correct.

**Reply by MiguelRipoll23** [reply]
That’s right, you should use the security system accessory itself to trigger the physical siren using an automation. The switch is just a way to trip the security system. Sorry for the confusing accessory name 😅
